_scripts/instruction-widget/install.js

```diff
@@ -116,6 +120,8 @@ module.exports = function(context) {
     context.package = "certbot"
     if (context.webserver == "apache") {
       context.package = "python-certbot-apache";
+    } else if (context.webserver == "nginx") {
+      context.certonly = true;
```

Comment:
it looks like it's called python-certbot-nginx in the ppa, https://launchpad.net/~certbot/+archive/ubuntu/certbot